appveyor.yml

```diff
@@ -1,4 +1,4 @@
-os: Windows Server 2012 R2
+os: Visual Studio 2015
```

Comment:
Yes, it's how AppVeyor works. See some more details at http://www.appveyor.com/updates.

Quite confusing, isn't it?